### DISCUSSION TITLE
Inspiration from around the world

### DISCUSSION THREAD

**Discussion by Ainali** [govdirectory/website#69] [Ideas]
This thread is meant to collect websites, designs, components and similar that can serve as an inspiration for this project. Please post anything that you find that could be useful here with a short explanation of what it is that you think we can learn from and why you think it might be useful to study,

**Comment by Ainali** [comment]
I'll start with the [US Digital Registry](https://usdigitalregistry.digitalgov.gov/) that seems to also provide a directory, but only for the US. 
Some interesting things: 
- how they use the amount in parenthesis already in dropdown lists for filters
- they have an API

Besides that, they are obviously also a great source for the US.

**Comment by Ainali** [comment]
The European Union [Social Networks](https://europa.eu/european-union/contact/social-networks_en) also have a similar functionality as this page.
To note:
- they lead with filter by platforms with icons
- they can stack/nest filters

This would potentially also be a great source, but unfortunately the results seem to be broken.

**Comment by Ainali** [comment]
The [Australian Governement Directory](https://www.directory.gov.au/) is an example of a directory with a lot of basic information and traditional contact information, but not any online accounts.

Besides all basic facts, they also have information and official contact details to the people in them.

**Comment by Ainali** [comment]
The [Swedish Agency Register](http://www.myndighetsregistret.scb.se/Myndighet) is similar to the Australian one but has less basic information and no information about the people. Of note though is that this is exempt from copyright and data can be freely be imported to Wikidata.

**Comment by Abbe98** [comment]
Ran into embassypages.com yesterday.

**Comment by Abbe98** [comment]
https://databook.wegov.nyc/organizations/chart

**Comment by RVA2869** [comment]
[https://consular-protection.ec.europa.eu/representation-offices_en](https://consular-protection.ec.europa.eu/representation-offices_en) and [https://www.embassy-worldwide.com/](https://www.embassy-worldwide.com/)

**Comment by RVA2869** [comment]
[https://government.github.com/community/](https://government.github.com/community/) Most of them are government related.

**Comment by Abbe98** [comment]
I just ran into this guide: https://united24media.com/war-in-ukraine/russia-is-relentlessly-bombing-kharkiv-heres-how-you-can-ask-for-more-air-defence-for-ukraine-35

which gave me the idea of "campaigns", curated list of government agencies and what to write in regard to a particular topic.

**Comment by RVA2869** [comment]
I came across this one on Reddit [Lowy Institute Global Diplomacy Index 2024](https://globaldiplomacyindex.lowyinstitute.org/)
And a well-structured data export is available. [https://globaldiplomacyindex.lowyinstitute.org/about](https://globaldiplomacyindex.lowyinstitute.org/about)
